src/shelly_light_controller.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Shelly-HomeKit Contributors
+ * All rights reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "shelly_light_controller.hpp"
+
+namespace shelly {
+
+LightController::LightController(struct mgos_config_lb *cfg, Output *out_w)
```

Comment:
This component is unused in this PR, it's just here because this PR is based on a prework wich I have made for #885, so it will be used for that PR for the Shelly Vintage support.

Reply:
it is used for the 4xW?